EIPS/eip-hard-gas-target.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: xxxx
+title: Hardcoded Block Gas Target
+author: Philippe Castonguay (@PhABC)
+discussions-to: xxxx
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-13
+
+---
+
+## Simple Summary
+
+Hardcode the block gas target (previously known as *block gas limit* before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)) to `12,500,000` gas per block.
```

Comment:
Since this EIP does not require 1559, I recommend using non-1559 terminology "gas limit".

Reply:
Good point. I'll leave a note about EIP-1559 renaming